docs/build/voting-dapp-tutorial.md

```diff
@@ -0,0 +1,1252 @@
+---
+title: dApp Tutorial
+description: Build a complete dApp on Kadena
+---
+
+import PageRef from '@components/PageRef'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Building a voting dApp
+
+In its most basic form, Kadena blockchain is a digital ledger. This ledger is never stored, but rather exists on the “chain” supported by thousands of nodes simultaneously. Thanks to encryption and decentralization, Kadena blockchain’s database of transactions is incorruptible, and each record is easily verifiable. The network cannot be taken down or influenced by a single party because it doesn’t exist in one place.
```

Comment:
I think there needs to be an introductory section to the tutorial. The current introduction in `Building a voting dApp` explains why the example app being built is important, which is good to include. But I think if someone is coming to this tutorial, they might not care too much about the app itself, and instead they might care more about what the different components of this end-to-end tutorial will be and why each component is important.

Reply:
I think you're right about not caring much about the app itself but learning the process of building **any app** on Kadena